system/Autoloader/FileLocator.php

```diff
@@ -36,372 +36,757 @@
  * @filesource
  */
 
+use Config\Services;
+use CodeIgniter\CLI\CLI;
+use CodeIgniter\Config\BaseConfig;
+use CodeIgniter\Exceptions\ConfigException;
+
 /**
- * Class FileLocator
- *
- * Allows loading non-class files in a namespaced manner.
- * Works with Helpers, Views, etc.
- *
- * @package CodeIgniter
+ * Class MigrationRunner
  */
-class FileLocator
+class MigrationRunner
```

Comment:
This file is the FileLocater file - and looks like you've copied MigrationRunner on it?

Reply:
Yeah I'm not even sure how I did that... Just sent over a fix. This PR has become kinda a mess, let me know if you'd like a fresh start.